shared-test-resources/TypeResolverTestSuite/ByteCodeHierarchyTest/src/Class1.java

```diff
@@ -0,0 +1,10 @@
+public class Class1 implements RootInterface1, SubInterface1{
+    @Override
+    public void method1(){
+        System.out.println("I'm method1 in Class1");
```

Comment:
I think adding println to the test case will slow the test down. is this needed?

Reply:
not necessarily - lets check which classes are added to the view and get into the typehierarchy

Reply:
I tend to use 
```
public int method1(){
 return 1;
}
```
for my empty methods

Reply:
if the test will create a call graph, it will be super slow.

Reply:
If the println is not needed for the test itself we should remove it.
it looks like the author has used it to execute the built test ressource

Reply:
it just creates a typehierarchy so its about~3s